app/views/clears-data.html

```diff
@@ -0,0 +1,34 @@
+
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+  Clear data? | GOV.UK Prototype Kit
+{% endblock %}
+
+{% block content %}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      {{ govukBackLink({
+          "text": "Back",
+          "href": "javascript: window.history.go(-1)"
```

Comment:
Non-blocking comment:

I believe we can populate this href using the referrer in the server, this is more effort and could be done as a followup.